wiki-service/src/main/java/org/exoplatform/wiki/service/search/SearchResult.java

```
@@ -76,7 +77,12 @@ public void setExcerpt(String text) {
   }
 
   public String getExcerpt() {
-    return Utils.sanitize(excerpt);
+    try {
+      return HTMLSanitizer.sanitize(excerpt);
+    } catch (Exception E) {
+

```

No empty catch block. You should add an error log here instead.